compiler/src/dotty/tools/backend/jvm/BCodeSyncAndTry.scala

```diff
@@ -406,11 +410,11 @@ trait BCodeSyncAndTry extends BCodeBodyBuilder {
     }
 
     /* Does this tree have a try-catch block? */
-    def mayCleanStack(tree: Tree): Boolean = tree exists { t => t match {
+    def mayCleanStack(tree: Tree): Boolean = tree.find { t => t match {
```

Comment:
Looks like there's `existsSubTree` that could be used here.

Reply:
Added TODO comment